frontend/app/module/polling_stations/page/PollingStationCreatePage.tsx

```diff
@@ -9,7 +9,7 @@ import { useNumericParam } from "@kiesraad/util";
 export function PollingStationCreatePage() {
   const electionId = useNumericParam("electionId");
   const navigate = useNavigate();
-  const onSave = (ps: PollingStation) => {
+  const handleSaved = (ps: PollingStation) => {
     navigate(`../?created=${ps.id}`);
```

Comment:
At this moment, we don't have an ID, but let's implement this in the validation issue https://github.com/kiesraad/abacus/issues/638

Reply:
This is not working for me, same issue that @jorisleker is mentioning

Reply:
Same here in Safari

Reply:
@lkleuver What's the status of this thread? What @oliver3 said, i.e. will be done in issue #638?

Reply:
the undefined came from the assumption of the backend returning the updated polling station, I've changed this to use the param from react router in PollingStationUpdatePage so the undefined is now defined.